aksparse/client/aksparse/sparse.py

```diff
@@ -105,92 +118,95 @@ def __mul__(self, other):
     def __rmul__(self, other):
         return self._binop_r(other, "*")
 
-    def plus_scalar(self, scalar):
+    def __plus_scalar(self, scalar):
         return
 
-    def sub_scalar(self, scalar):
+    def __sub_scalar(self, scalar):
         return
 
-    def mul_scalar(self, scalar):
+    def __mul_scalar(self, scalar):
         return
 
-    def plus_vector(self, vector):
+    def __plus_vector(self, vector):
         return
 
-    def sub_vector(self, vector):
+    def __sub_vector(self, vector):
         return
 
-    def mul_vector(self, vector):
+    def __mul_vector(self, vector):
         return
```

Comment:
Are you intending for these to be class attributes? Or are they intended to be private functions? If we are doing the later (which based on usage I believe we are), only a single underscore should be used.

Reply:
Yes, they were meant to be private functions, but I was mistaken on the syntax for that